portfolio/src/main/webapp/script.js

```diff
@@ -30,3 +30,20 @@ function addRandomFunFact() {
   const factContainer = document.getElementById('fact-container');
   factContainer.innerText = fact;
 }
+
+var i = 1;
+var projects = ["images/proj1.png","images/proj2.png","images/proj3.png","images/proj4.png"];
+var time = 3000;
```

Comment:
changeProjectTimeMs?

`time` alone is ambiguous since it's not clear what the time should be used for. The Ms suffix will help a reader understand this is meant to be milliseconds.